src/views/Game/Game.styl

```diff
@@ -397,6 +397,7 @@ goban-view-bar-width=400px
         align-items: stretch;
         width: 100%;
 
+        // Why? ...
```

Comment:
Haha [I just delete these when I see them](https://github.com/online-go/online-go.com/pull/1383/commits/e395201c3d24bd7277895343ff3dc55a78525577)... if there's a reason for them, I'd like to know as well!

Reply:
Yeah those are always safe to delete, that's just slop - looks like that came in during the initial import into the open source repo 4 years ago - slop.

Reply:
We still love you anoek-senpai ❤ thanks for your hard work!